src/services/CertificateGenerationService.ts

```diff
@@ -98,42 +83,39 @@ class CertificateGenerationService {
       }),
     };
     return this.lambdaClient
-      .invoke(invokeParams)
-      .then(
-        (
-          response: PromiseResult<Lambda.Types.InvocationResponse, AWSError>
-        ) => {
-          const documentPayload: any =
-            this.lambdaClient.validateInvocationResponse(response);
-          const resBody: string = documentPayload.body;
-          const responseBuffer: Buffer = Buffer.from(resBody, "base64");
-          return {
-            vrm:
-              testResult.vehicleType === VEHICLE_TYPES.TRL
-                ? testResult.trailerId
-                : testResult.vrm,
-            testTypeName: testResult.testTypes.testTypeName,
-            testTypeResult: testResult.testTypes.testResult,
-            dateOfIssue: moment(
-              testResult.testTypes.testTypeStartTimestamp
-            ).format("D MMMM YYYY"),
-            certificateType: certificateTypes[vehicleTestRes].split(".")[0],
-            fileFormat: "pdf",
-            fileName: `${testResult.testTypes.testNumber}_${testResult.vin}.pdf`,
-            fileSize: responseBuffer.byteLength.toString(),
-            certificate: responseBuffer,
-            certificateOrder: testResult.order,
-            email:
-              testResult.createdByEmailAddress ?? testResult.testerEmailAddress,
-            shouldEmailCertificate: testResult.shouldEmailCertificate ?? "true",
-          };
-        }
-      )
-      .catch((error: AWSError | Error) => {
-        console.log(error);
-        throw error;
-      });
-  }
+            .invoke(invokeParams)
+            .then(
+                async (response: PromiseResult<Lambda.Types.InvocationResponse, AWSError>) => {
+                    const documentPayload: any = await this.lambdaClient.validateInvocationResponse(response);
+                    const resBody: string = documentPayload.body;
+                    const responseBuffer: Buffer = Buffer.from(resBody, "base64");
+                    return {
+                        vrm:
+                            testResult.vehicleType === VEHICLE_TYPES.TRL
+                                ? testResult.trailerId
+                                : testResult.vrm,
+                        testTypeName: testResult.testTypes.testTypeName,
+                        testTypeResult: testResult.testTypes.testResult,
+                        dateOfIssue: moment(
```

Comment:
Might be worth considering whether or not to replace moment with luxon

Reply:
I think for the purpose of the ticket, might be worth leaving it out for now and bump the dependencies in another ticket